### PR TITLE
Migration to new cyclonedx-python-lib for SBOM generation

### PR DESCRIPTION
**Pull request by madpah**

### CODE REVIEW

**Comment by madpah** [comment]
@stevespringett @DarthHater - PR to update `cyclonedx-python` to use the new `cyclonedx-python-lib`.

Feedback appreciated.

File: setup.py [hunk 1]
```diff
@@ -1,40 +1,26 @@
...
     maintainer='Steve Springett',
     maintainer_email='steve.springett@owasp.org',
-    url='https://github.com/CycloneDX/cyclonedx-python',
+    description='CycloneDX Software Bill of Materials (SBOM) generation utility',
+    long_description=open(os.path.join(script_path, 'README.md')).read(),
+    long_description_content_type="text/markdown",
     keywords=["BOM", "SBOM", "SCA", "OWASP"],
     license='Apache-2.0',
     classifiers=[
+        'Development Status :: 4 - Beta',
```
Comment (jkowalleck) [inline setup.py:23]: that is odd.
 here it is marked as beta... 

version is set to `1.0.0` in `VERSION ` file 
in the `pyproject.toml` its `1.0.0`, too 

it this was a beta, then the version should be `1.0b.0` according to 
PEP 440 - https://www.python.org/dev/peps/pep-0440/#pre-releases

**Review by jkowalleck — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (madpah) [reply setup.py]: Updated - thanks @jkowalleck

**Review by jkowalleck — APPROVED** [review]

**Comment by jkowalleck** [comment]
unfortunately a review from a maintainer is required.
from my side everything seams to be fine. i mot a known maintainer of this repo.

@coderpatros do you know a maintainer of this repo who can review (and approve) and merge this PR?

**Comment by stevespringett** [comment]
Huge thanks for the PR

**Pull request merged**